### DISCUSSION TITLE
Difference to viz.js

### DISCUSSION THREAD

**Discussion by septatrix** [hpcc-systems/hpcc-js-wasm#289] [Q&A] [answered]
The graphviz module in this repo and viz.js seem to do pretty much the same. Are there any known differencens? E.g. what parts of graphviz are supported or how exactly they are compiled?

As this project is newer I suppose that some parts of viz.js were insufficient for your needs and you therefore created this module which is why I am asking here. If you do not know, feel free to just close this

**Comment by GordonSmith** [comment, marked as answer]
1. Viz.js was not being updated for a long time and I wanted to use newer versions of the GraphViz sources (I was previously using GraphViz in NAPI / ActiveX type environments), so I rolled my own build chain.  Its only fairly recently that Viz.js started to package the latest and greatest versions again (and I must confess that the first time I saw Viz.js in action using asm js I was seriously impressed!).
2. I may be wrong, but I don't think Viz.js supports the html tables feature, for things like this:  https://observablehq.com/@gordonsmith/graphviz 
3. I was also interested in other libs beyond GraphViz so spent a fair amount of time tweaking the package layouts and build pattern to a point I was happy with...
